ReactSkia/textlayoutmanager/RSkTextLayoutManager.cpp

```diff
@@ -47,6 +47,14 @@ SkPaint convertTextColor ( SharedColor textColor ) {
    return paint;
 }
 
+SkColor convertTextPropColor ( SharedColor textColor ) {
```

Comment:
convertTextColor and convertTextPropColor is doing same functionality, can we use already available convertTextColor() API itself. This function is probably not required.

Reply:
Updated